### ISSUE TITLE
Uncaught TypeError: this.triggerElement is null

### ISSUE THREAD

**Issue by wiryonolau** [VizuaaLOG/BulmaJS#138]
**Versions**
BulmaJS: 0.12.1
Browser: Firefox 96
Operating system: Ubuntu 18.04

**Describe the bug**
On load console will report
```
Uncaught TypeError: this.triggerElement is null
```

**To Reproduce**
Load with ES5 

```html
<link href="/bulma/css/bulma.min.css" media="screen" rel="stylesheet" type="text/css" >
<script type="text/javascript" defer="defer" src="/@vizuaalog/bulmajs/dist/bulma.js"></script>
```

**Comment by wiryonolau** [comment]
I found the problem in the navbar.js

Since its automatically modify the navbar, I need to make sure put a complete navbar element. Any missing component will get the error

This minimum is required
```
nav.navbar
    div.navbar-brand
         a.navbar-burger
    div.navbar-menu
```

Example
```
<nav class="navbar" role="navigation" aria-label="main navigation" data-bulma-attached="attached">
    <div class="navbar-brand">
        <a class="navbar-item" style="margin: 0 10px">
             BRAND
        </a>
        <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
             <span aria-hidden="true"></span>
             <span aria-hidden="true"></span>
            <span aria-hidden="true"></span>
        </a>
    </div>
    <div class="navbar-menu">
        <div class="navbar-start">
        </div>
    </div>
</nav>
```

It fixes for me, but maybe you can give a log if element not exist or skip converting

**Comment by VizuaaLOG** [comment]
Happy to accept a PR to add a more helpful error if the element is missing. I don't use this library or Bulma anymore. Otherwise, I can implement this when I have some time to work on it.

**Comment by VizuaaLOG** [comment]
Fixed in 0.12.2
